iset.mm

```diff
@@ -19416,6 +19448,18 @@ This law is thought to have originated with Aristotle (_Metaphysics_,
       EHZPIQNIABECDJOECKAEBKLM $.
   $}
 
+  ${
+    $d x y z $.  $d ph z $.  $d ps z $.
+    cbvabw.1 $e |- F/ y ph $.
+    cbvabw.2 $e |- F/ x ps $.
+    cbvabw.3 $e |- ( x = y -> ( ph <-> ps ) ) $.
+    $( Version of ~ cbvab with a disjoint variable condition.  (Contributed by
+       Gino Giotto, 10-Jan-2024.) $)
+    cbvabw $p |- { x | ph } = { y | ps } $=
+      ( vz cab wsb cv wcel sbco2v sbiev sbbii bitr3i df-clab 3bitr4i eqriv ) HA
+      CIZBDIZACHJZBDHJZHKZTLUDUALUBACDJZDHJUCACHDEMUEBDHABCDFGNOPAHCQBHDQRS $.
+  $}
+
```

Comment:
Regarding such comments, see discussion in #3879 and #3905. These comments should also be aligned in iset.mm, after the discussion leads to changes in set.mm.